src/AWS.Deploy.Recipes/RecipeDefinitions/ASP.NETAppElasticBeanstalk.recipe

```diff
@@ -73,11 +73,63 @@
             }
         }
     ],
-
+    "Categories": [
+        {
+            "Id": "General",
+            "DisplayName": "General",
+            "Order": 10
+        },
+        {
+            "Id": "Hosting",
+            "DisplayName": "Hosting",
+            "Order": 20
+        },
+        {
+            "Id": "Platform",
+            "DisplayName": "Platform",
+            "Order": 30
+        },
+        {
+            "Id": "RollingUpdates",
+            "DisplayName": "Rolling updates and deployments",
```

Comment:
nit; to be consistent, could you use & instead of "and"?

Reply:
Done